### DISCUSSION TITLE
connection fails from bitwarden official android app

### DISCUSSION THREAD

**Discussion by mauroruffino** [dani-garcia/vaultwarden#4668] [General]
hi everybody
im new in this git repo discussions 

i install vaultwarden in a zimaboard casaos container and from windows official bitwarden app and from browser all works perfectly fine

but from bitwarden official android app i dont connect it in selfhosted, app return a generic connection fail

i use https;//.... url that works from windows app and works from browser but dont works from android

in docker container i use nginx and letsencryp certificate , its possible that certificate dont works in my android phone ?

can anyone helps me

thanks bye
mauro

**Comment by likevi9528** [comment]
Whats your Android app version？The newest version (beta.2024.6.1) doesn't support the vaultwarden #4656 #4386 .
Use lets certifications may not cause errors.

**Reply by mauroruffino** [reply]
11, but I make a mistake in url used in bitwarden selfhelp app, now all it works, sorry
